src/Core/SettingsChangesHistory.cpp

```diff
@@ -98,6 +98,7 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
             {"validate_enum_literals_in_operators", false, false, "A new setting"},
             {"allow_experimental_kusto_dialect", true, false, "A new setting"},
             {"allow_experimental_prql_dialect", true, false, "A new setting"},
+            {"h3togeo_standard_order", false, true, "New setting, fixes h3ToGeo() function to return (lat, lon)"},
```

Comment:
"standard_order" sounds slightly weird. Maybe "result_order"?

EDIT 1: Oh, I see "result_order" would not work either because the parameter is boolean. Can we rename it to "h3togeo_lat_lon_result_order"

EDIT 2: We should invert the semantics. The database ideally has as _few_ settings enabled by default as possible. Hence, the new default value for the setting should be `false`. If users like to go back to the previous behavior, they should need to enable a setting (not the other way round: disable a setting). Can we name the setting `h3togeo_lon_lat_result_order` (default: false)?

Reply:
Fixed.